pyproject.toml

```diff
@@ -103,9 +103,6 @@ omit = ["*/tests/*"]
 plugins = ["covdefaults"]
 source = ["type_lens"]
 
-[tool.coverage.report]
```

Comment:
the actual config name is `fail_under`, so this wasn't doing anything but causing warnings when running `make coverage`. But also it's not possible to get 100% coverage inside a given python version. So realistically, this can only be made to have 100% coverage at the CI level.